css/cart.css

```diff
@@ -0,0 +1,107 @@
+.btn-remove{
+    background-color: var(--white);
+    border:1px solid var(--light-red2);
+    color: var(--light-red2);
+}
+
+.btn-remove:hover{
+    background-color: var(--light-red2);
+    color: var(--white);
+}
+
+.quantiy-section{
+    display: flex;
+    margin: 1rem 0rem;
```

Comment:
0 values don't need a unit

Reply:
Thanks for pointing that out. I have made the changes.